web-components/chart/chart.tsx

```diff
@@ -212,7 +246,13 @@ export class CobChart {
         // it allows us to use Vega transforms then use the transformed data
         // to populate our selections. It also means we don't have to
         // fetch our data twice - Vega does it once and we leverage it.
-        this.dataset = this.view.data(this.config.data.name);
+        this.dataset = this.vegaLite
```

Comment:
Is this good to condition on `this.vegaLite`, or just whether or not `this.config.data` is an array or object?

Reply:
I feel like for readability conditioning on vegaLite makes sense to me - if I condition on `this.config.data` being an array or object I think it makes it less clear to the reader why the condition exists. The condition on vegaLite makes it clear there is a difference in the way the configs store the data vs. a reader having to know that about Vega and VegaLite to understand where the statement is coming from. 

I left it the same in the commit I just pushed but can always add a comment explaining, so lmk if you feel strongly.

Reply:
OK, sounds good!